src/components/elements/MobileNav.jsx

```diff
@@ -64,7 +64,7 @@ function MobileNav() {
             <Link to="/events/new">Post Event</Link><br/>
             <Link to="/events/weekly-digest">Weekly Digest</Link><br/>
             <Link to="/sponsors">Sponsors</Link><br/>
-            <Link to="/donate">Donate</Link><br/>
+            <a href="https://opencollective.com/data-umbrella/contribute/data-science-event-board-37473/checkout">Donate</a><br/>
```

Comment:
Not blocking, but we should pull this URL into a constant that we can import when needed to remove duplication.